h/api/models.py

```diff
@@ -47,6 +48,7 @@ def __acl__(self):
         'tags': {'type': 'string', 'analyzer': 'uni_normalizer'},
         'text': {'type': 'string', 'analyzer': 'uni_normalizer'},
         'deleted': {'type': 'boolean'},
+        'nipsa': {'type': 'boolean'},
```

Comment:
This is the only thing in this code that seemed out of place, but I guess this is just exposing the `nipsa` flag on the annotation so UI-side code can Do The Right Thing with it?